omod-2.2/pom.xml

```diff
@@ -19,6 +19,28 @@
 			<groupId>${project.parent.groupId}</groupId>
 			<artifactId>${project.parent.artifactId}-omod-common</artifactId>
 			<version>${project.parent.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
hacky but i needed to add these exclusions because the jackson related dependencies coming from swagger-core 2.2.23 are quite high